package.json

```diff
@@ -35,13 +35,13 @@
     "axios": "~0.21.0",
     "expo": "~42.0.0",
     "expo-application": "~3.2.0",
+    "expo-cli": "^6.0.6",
     "expo-constants": "~11.0.1",
     "expo-device": "~3.3.0",
     "expo-linking": "~2.3.1",
     "expo-secure-store": "~10.2.0",
     "expo-status-bar": "~1.0.4",
     "expo-updates": "~0.8.5",
-    "font": "~0.0.4",
```

Comment:
The license for this tool cannot be determined since the source code is no longer available and I didn't find any forks. It doesn't seem to be used so I removed it.